docs/src/modules/java/pages/views.adoc

```diff
@@ -408,6 +408,38 @@ include::example$scala-eventsourced-customer-registry/src/main/scala/customer/Ma
 
 The View definitions are stored and validated when a new version is deployed. There will be an error message if the changes are not compatible.
 
+== Streaming view updates
+
+A query can provide a near real time stream of results for the query, emitting new entries matching the query as they are added or updated in
+the view.
+
+This is done by adding the option `stream_updates` to a query method with a `stream` reply:
+
+[.tabset]
+Java::
++
+[source,proto,indent=0]
+----
+include::example$java-eventsourced-customer-registry/src/main/proto/customer/view/customer_view.proto[tag=stream-updates]
+----
+<1> A regular query, in this case showing all customers from a specific city
+<2> The `stream_updates` option set to `true`
+
+Scala::
++
+[source,proto,indent=0]
+----
+include::example$scala-eventsourced-customer-registry/src/main/proto/customer/view/customer_view.proto[tag=stream-updates]
+----
+<1> A regular query, in this case showing all customers from a specific city
+<2> The `stream_updates` option set to `true`
+
+This will first list the complete result for the query and then keep the response stream open, emitting new or updated
+entries matching the query as they are added to the view. The stream does not complete until the client closes it.
+
+Note: This is not intended as transport for service to service propagation of updates and does not guarantee delivery, for such use cases you
+should instead publish events to a topic, see xref:actions-publishing-subscribing.adoc[Publishing and Subscribing with Actions]
```

Comment:
I found this NOTE to be very insightful. I wonder I fully understand it, though. 
- `service to service propagation` refers only to AkkaServerless services, right?  
- `for such use cases you should instead publish` is because of the delivery guarantees?

Reply:
Any service, Akka serverless or implemented with some other tech, should use message broker rather than this, for delivery guarantees and decoupling (a broker will still have the messages when the sending service is offline, and can still take published messages when the consumer is offline)